lumigator/backend/backend/api/router.py

```diff
@@ -12,3 +12,4 @@
 api_router.include_router(experiments.router, prefix="/experiments", tags=[Tags.EXPERIMENTS])
 api_router.include_router(models.router, prefix="/models", tags=[Tags.MODELS])
 api_router.include_router(workflows.router, prefix="/workflows", tags=[Tags.WORKFLOWS])
+api_router.include_router(secrets.router, prefix="/settings/secrets", tags=[Tags.SETTINGS])
```

Comment:
Maybe......
```suggestion
api_router.include_router(secrets.router, prefix="/settings/secrets", tags=[Tags.SECRETS])
```
This is only meant for swagger docs grouping, so the current `SETTINGS` would also be fine to me.

Reply:
I wonder if we should leave it as `SETTINGS` for now as it covers the higher-level grouping and then as you say in Swagger docs it means if we add other settings related endpoints they'll just appear together?